### PR TITLE
Rename Vector4f message to Quaternion

### PR DESCRIPTION
**Pull request by ksavinash9**
Rename `Vector4f` message to `Quaternion`

### CODE REVIEW

**Comment by ksavinash9** [comment]
@SirVer please review.

File: point_viewer_grpc_proto_rust/src/proto.proto [hunk 1]
```diff
@@ -52,7 +52,7 @@ message GetPointsInBoxRequest {
 
 message GetPointsInFrustumRequest {
   point_viewer.proto.Vector3f translation = 1;
-  point_viewer.proto.Vector4f rotation = 2;
+  point_viewer.proto.Quaternion rotation = 2;
```
Comment (SirVer) [inline point_viewer_grpc_proto_rust/src/proto.proto]: For consistency call this `Quaternionf` instead of just `Quaternion`?

**Comment by ksavinash9** [comment]
@wally-the-cartographer merge

**Comment by ksavinash9** [comment]
@SirVer please merge.

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by schwoere** [comment]
@wally-the-cartographer merge

**Pull request merged**